static/app/components/charts/optionSelector.tsx

```diff
@@ -58,12 +60,15 @@ class OptionSelector extends Component<Props, State> {
 
   render() {
     const {menuContainerWidth} = this.state;
-    const {options, onChange, selected, title, menuWidth} = this.props;
+    const {options, onChange, selected, title, menuWidth, featureType} = this.props;
     const selectedOption = options.find(opt => selected === opt.value) || options[0];
 
     return (
       <InlineContainer>
-        <SectionHeading>{title}</SectionHeading>
+        <SectionHeading>
+          {title}
+          {featureType ? <StyledFeatureBadge type={featureType} /> : null}
```

Comment:
I think, just to be explicit, you can do `defined(featureType)`, it's not a big deal here though since it has to be one of the three options or undefined